extensions/amp-consent/0.1/cmps.js

```diff
@@ -76,3 +76,8 @@ CMP_CONFIG['Usercentrics'] = {
   'checkConsentHref': 'https://consents.usercentrics.eu/amp/checkConsent',
   'promptUISrc': 'https://amp.usercentrics.eu/amp.html',
 };
+
+CMP_CONFIG['iubenda'] = {
+  'consentInstanceId': 'iubenda',
+  'checkConsentHref': 'https://hits.iubenda.com/cs/amp/checkConsent',
+};
```

Comment:
nit: place after didomi

Reply:
> Hi, thank you for your integration and sorry about the late review!
> 
> Left changes to be made (don't forget to make the same changes to in cmp-vendors to your example documentation).

Thanks for your review. Can you please read the PR description as we have made some clarifications and made also a question? Thanks!

Reply:
Done.